Uchu.StandardScripts/NimbusStation/GetFaction.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+using Uchu.Core.Client;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+using Uchu.World.Systems.Missions;
+
+namespace Uchu.StandardScripts.NimbusStation
+{
+    [ZoneSpecific(1200)]
+    public class GetFaction : NativeScript
+    {
+        int MissionID { get; set; } = 474;
+
+        public override Task LoadAsync()
+        {
+            Listen(Zone.OnPlayerLoad, player => {
+                Listen(player.OnRespondToMission, async (missionID, playerObject, rewardItem)  => {
+                    if (missionID != MissionID) return;
+
+                    if (rewardItem.Id == -1) return; // If no reward chosen 
+
+                    int[] Missions = new int[3];
+                    int CelebrationID = -1;
+                    int FactionFlag = 0;
+
+                    if (rewardItem.Id == 6980) // Venture
```

Comment:
Please use resources here as well

Reply:
Missions, celebrations and flags, that is.

Reply:
Ok I will do that but I do want to investigate what Wincent said about the build scripts.

Reply:
For the record: Wincent mentioned using code gen based on the cd client.